index.html

```diff
@@ -0,0 +1,196 @@
+              applied science.
+            </span>
+          </li>
+          <li><span class="date">1942</span> - <span class="detail"></span></li>
+          <li>
+            <span class="date"></span
+            ><span class="detail"
+              >Receives a Ph.D. in Genetics and Plant Pathology
+            </span>
+          </li>
+          <li>
+            <span class="date">1944</span> -
+            <span class="detail"
+              >Rejects a 100% salary increase from Dupont, leaves behind his
+              pregnant wife, and flies to Mexico to head a new plant pathology
+              program. Over the next 16 years, his team breeds 6,000 different
+              strains of disease resistent wheat - including different varieties
+              for each major climate on Earth.
+            </span>
+          </li>
+          <li>
+            <span class="date">1945</span> -
+            <span class="detail"
+              >Discovers a way to grown wheat twice each season, doubling wheat
+              yields
+            </span>
+          </li>
+
+            <span class="date">1953</span>
+            <span class="date">1953</span>
+            <span class="detail"
+              >crosses a short, sturdy dwarf breed ofwheat with a high-yeidling
+              American breed, creating a strain that responds well to
+              fertilizer. It goes on to provide 95% of Mexico's wheat</span
+            >
+          </li>
+          <li>
+            <span class="date">1962</span> -
+            <span class="detail"
+              >Visits Delhi andrings his high-yielding strains of wheat to the
+              Indian subcontinent in time to help mitigate mass starvation due
+              to a rapidly expanding population
+            </span>
+          </li>
+          <li>
+            <span class="date">1970</span> -
+            <span class="detail">receives the Nobel Peace Prize</span>
+          </li>
+          <li>
+            <span class="date">1983</span> -
+            <span class="detail"
+              >helps seven African countries dramatically increase their maize
+              and sorghum yields
+            </span>
+          </li>
+          <li>
+            <span class="date">1984</span> -
+            <span class="detail"
+              >becomes a distinguished professor at Texas A&M University
+            </span>
+          </li>
+          <li>
+            <span class="date">2005</span> -
+            <span class="detail"
+              >states "we will have to double the world food supply by 2050."
+              Argues that genetically modified crops are the only way we can
+              meet the demand, as we run out of arable land. Says that GM crops
+              are not inherently dangerous because "we've been genetically
+              modifying plants and animals for a long time. Long before we
+              called it science, people were selecting the best breeds."
```

Comment:
add `p` tag

Reply:
> add `p` tag

You did great for the first review about the use of `p` tag to ensure the accessibility 👏 . You also used polite language to suggest and gave a brief explanation `why`. But i suggest that the same review approach is used here. This will make the inline comment more meaningful to the project contributor(s) ☺️

Reply:
Kindly add a p tag in place of span tags. As mentioned previously p tags enable differentiation of the text levels optimizing SEO +1 :+1: